youtube_dl_gui/locale/es_ES/LC_MESSAGES/youtube_dl_gui.po

```diff
@@ -0,0 +1,440 @@
+msgstr "General"
+
+#: optionsframe.py:46
+msgid "Video"
+msgstr "Video"
+
+#: optionsframe.py:47
+msgid "Audio"
+msgstr "Audio"
+
+#: optionsframe.py:48
+msgid "Playlist"
+msgstr "Lista de reproduccion"
+
+#: optionsframe.py:49
+msgid "Output"
+msgstr "Salida"
+
+#: optionsframe.py:50
+msgid "Subtitles"
+msgstr "Subtitulos"
+
+#: optionsframe.py:51
+msgid "Filesystem"
+msgstr "Sistema de archivos"
+
+#: optionsframe.py:52
+msgid "Shutdown"
+msgstr "Apagar"
+
+#: optionsframe.py:53
+msgid "Authentication"
+msgstr "Autenticación"
+
+#: optionsframe.py:54
+msgid "Connection"
+msgstr "Conexion"
+
+#: optionsframe.py:55
+msgid "Log"
+msgstr "Registro"
+
+#: optionsframe.py:56
+msgid "Commands"
+msgstr "Comandos"
+
+#: optionsframe.py:57
+msgid "Localization"
+msgstr "Localización"
+
+#: optionsframe.py:375
+msgid "Enable Log"
+msgstr "Permitir registro"
+
+#: optionsframe.py:376
+msgid "Write Time"
+msgstr "Escribir fecha"
+
+#: optionsframe.py:377
+msgid "Clear Log"
+msgstr "Limpiar registro"
+
+#: optionsframe.py:378
+msgid "View Log"
+msgstr "Ver registro"
+
+#: optionsframe.py:379
+msgid "Path: {0}"
+msgstr "Ruta: {0}"
+
+#: optionsframe.py:380
+msgid "Log Size: {0} Bytes"
+msgstr "Tamaño del registro: {0} Bytes"
+
+#: optionsframe.py:381 optionsframe.py:1394
+msgid "Restart"
+msgstr "Reiniciar"
+
+#: optionsframe.py:382
+msgid "Please restart {0}"
+msgstr "Por favor reinicie {0}"
+
+#: optionsframe.py:490
+msgid "Shutdown when finished"
+msgstr "Apagar cuando termine"
+
+#: optionsframe.py:491
+msgid "SUDO password"
+msgstr "Contraseña de SUDO"
+
+#: optionsframe.py:543
+msgid "Playlist Start"
+msgstr "Comenzar lista de reproduccion"
+
+#: optionsframe.py:544
+msgid "Playlist Stop"
+msgstr "Detener lista de reproduccion"
+
+#: optionsframe.py:545
+msgid "Max Downloads"
+msgstr "Descargas maximas"
+
+#: optionsframe.py:603
+msgid "Retries"
+msgstr "Reintentos"
+
+#: optionsframe.py:604
+msgid "User Agent"
+msgstr "Agente de Usuario"
+
+#: optionsframe.py:605
+msgid "Referer"
+msgstr "Referido"
+
+#: optionsframe.py:606
+msgid "Proxy"
+msgstr "Proxy"
+
+#: optionsframe.py:680
+msgid "Username"
+msgstr "Usuario"
+
+#: optionsframe.py:681
+msgid "Password"
+msgstr "Contraseña"
+
+#: optionsframe.py:682
+msgid "Video Password (vimeo, smotri)"
+msgstr "Contraseña del vídeo (vimeo, smotri)"
+
+#: optionsframe.py:740
+msgid "high"
+msgstr "alta"
+
+#: optionsframe.py:740
+msgid "low"
+msgstr "baja"
+
+#: optionsframe.py:740
+msgid "mid"
+msgstr "media"
+
+#: optionsframe.py:743
+msgid "Convert to Audio"
+msgstr "Convertir a Audio"
+
+#: optionsframe.py:744
+msgid "Keep Video"
+msgstr "Mantener video"
+
+#: optionsframe.py:745
+msgid "Audio Format"
+msgstr "Formato de Audio"
+
+#: optionsframe.py:746
+msgid "Audio Quality"
+msgstr "Calidad del Audio"
+
+#: optionsframe.py:863
+msgid "default"
+msgstr "defecto"
+
+#: optionsframe.py:864
+msgid "none"
+msgstr "ninguno"
+
+#: optionsframe.py:868
+msgid "Video Format"
+msgstr "Formato del videio"
```

Comment:
On line 132 you translate Video as Video (no changes).
Here you write videio. Is this a typo?